src/java/org/apache/cassandra/config/GuardrailsOptions.java

```diff
@@ -1080,8 +1164,8 @@ private static void validateWarnLowerThanFail(long warn, long fail, String name)
             return;
 
         if (fail < warn)
-            throw new IllegalArgumentException(format("The warn threshold %d for %s_warn_threshold should be lower " +
-                                                      "than the fail threshold %d", warn, name, fail));
+            throw new IllegalArgumentException(format("The warn threshold %d for %s_warn_threshold should not be " +
```

Comment:
That should be ok. I doubt anybody is parsing these messages. Just mentioning it to be on the safe side.